### ISSUE TITLE
To Do

### ISSUE THREAD

**Issue by maria-yampolskaya** [maria-yampolskaya/ProjectOAK#2]
- [x]  Compare results to version without secondary types
- [x]  Apply pre-built neural net ([transfer learning](https://www.tensorflow.org/guide/keras/transfer_learning))
- [x] Use [Keras Tuner](https://www.tensorflow.org/tutorials/keras/keras_tuner)
- [x]  Save everything as jpgs

**Comment by Sevans711** [comment]
- [ ] Long-term goal: Multi-label network (see e.g. [here](https://towardsdatascience.com/multi-label-text-classification-with-scikit-learn-30714b7819c5))
- [x] Meet on Dec 1 at 2
- [x] Minimum for next time: make a neural network that does the basic thing we want to do
- [x] Data Augmentation (include rotated / stretched / faded copies of images in initial dataset.)
